### DISCUSSION TITLE
when installing the markdown language pack markdown preview fail

### DISCUSSION THREAD

**Discussion by andresparrab** [LazyVim/LazyVim#4410] [General]
When installed the markdown languages pack from Lazyentra, it  look like install everything exept the markdown-preview and i dont know what is th reason of the problem

![image](https://github.com/user-attachments/assets/e9b119e0-edc2-4527-82f1-cad710408099)

**Comment by dpetka2001** [comment]
This has already been fixed in HEAD https://github.com/LazyVim/LazyVim/commit/b892861bdee888f83401e745bda1798dc01e2c54. You just have to wait for the next LazyVim release or start using HEAD instead as per the [docs](https://www.lazyvim.org/configuration/lazy.nvim).

**Comment by andresparrab** [comment]
Thank you. I will wait until next release.. It is not that hurry.
I just started to use obsidian for note taking but look like many recommendations to use nvim as I already use it for
DotNet development so I hope start to write all my notes as a Second brain and do all the things I use in obsidian..

One thing I don't know how to do that I see many people do in nvim, is for example get auto suggestions to create like table of contents of all headers 
or refere to another document in same folder like [my title2](file2.md#title2)​ and it Will jump that file and section 
I can only jumt to the file if I refer it like [my title2](file2.md)​

**Reply by dpetka2001** [reply]
Sorry, but I don't do Markdown myself. Hopefully someone else can help you with what you're asking.

As for the `markdown-preview`, just create a file `~/.config/nvim/lua/plugins/markdown.lua` with contents
```lua
return {
 {
    "iamcco/markdown-preview.nvim",
    build = function()
      require("lazy").load({ plugins = { "markdown-preview.nvim" } })
      vim.fn["mkdp#util#install"]()
    end,
  },
}
```
This should allow you to build it right away without waiting for the release.
